packages/shared/src/lib/reminderFields.ts

```diff
@@ -41,9 +51,7 @@ export const buildReminderFields = (today: Date = new Date()): ReminderFields =>
     };
 };
 
-const givingTuesdayCutOff = new Date('2023-11-27');
-
-const givingTuesdayIsActive = (date: Date): boolean => date <= givingTuesdayCutOff;
+const givingTuesdayIsActive = (date: Date): boolean => date < givingTuesday();
```

Comment:
So during 2023 when we're past Giving Tuesday this will start returning false. When we tick over to 2024, won't this start returning true again for that year? I think to make this dynamic we also need a start cut off otherwise we'll return true from Jan 1 - the day before Giving Tuesday? (Apologies if I'm misunderstanding something here!)

Reply:
You're absolutely right. I expended most of my energy trying to figure out a neat way to calculate the date and didn't think of this! I'll check with PMs from what date we should start offering this reminder and make a change accordingly.